Rule.ts

```diff
@@ -69,7 +69,7 @@ export class Rule {
 
     private static parseInboxActionType(str: string): InboxActionType {
         if (str.length === 0) {
-            return InboxActionType.DEFAULT;
+            return InboxActionType.NOTHING;
```

Comment:
An alternative is that we [iterate over all the rule cells](https://developers.google.com/apps-script/reference/spreadsheet/data-validation#copy) and change the rules (if they don't already) to have `NOTHING` added to the DataValidation criteria values. Then just let someone select `NOTHING`.